guides/common/modules/proc_configuring-for-uefi-http-boot-ipv6-provisioning.adoc

```diff
@@ -21,7 +21,7 @@ ifndef::foreman-deb[]
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-update-project} grub2-efi
+# {package-manager} update grub2-efi
```

Comment:
I would rather change `package-update-project` in `guides/common/attributes-base.adoc` to `dnf update`.

Reply:
That get's tricky, as I believe that attribute is used in a variety of places where `dnf` is not appropriate, for example, some of the client documentation. I found other places where we are currently using `{package-manager} update` and so I opted to conform with that standard for these changes rather than having to tackle a much bigger problem.

Reply:
You're breaking things because in the Satellite docs the update command is actually `satellite-maintain package update`. The fact that on downstream it uses `satellite-maintain` proves that it's not intended for client documentation and it doesn't appear to be used anywhere except on the server:
```console
$ rg package-update-project
guides/common/attributes-foreman-deb.adoc
14::package-update-project: apt-get upgrade

guides/common/attributes-base.adoc
69::package-update-project: yum update

guides/common/attributes-satellite.adoc
65::package-update-project: satellite-maintain packages update

guides/doc-Planning_for_Project/topics/Provisioning_Concepts.adoc
88:# {package-update-project} grub2-efi
129:# {package-update-project} grub2-efi

guides/common/modules/proc_managing-packages-on-the-base-operating-system.adoc
23:# {package-update-project} _package_1_ _package_2_
29:# {package-update-project}

guides/common/modules/proc_creating-hosts-with-uefi-http-boot-provisioning.adoc
19:# {package-update-project} grub2-efi
95:# {package-update-project} grub2-efi

guides/common/modules/proc_configuring-for-uefi-http-boot-ipv6-provisioning.adoc
24:# {package-update-project} grub2-efi
```